test/utils.py

```diff
@@ -217,6 +219,185 @@ def all_files_exist():
     return wait_event(timeout, all_files_exist, interval)
 
 
+class TtApp(object):
+    # Helper type to represent target. Commands (start, stop, etc.) treat it as following:
+    # None - no target at all (ex: tt start)
+    # Target(None) - app name only (ex: tt start some_app)
+    # Target('some_inst') - app:inst target (ex: tt start some_app:some_inst)
+    Target = namedtuple('Target', ['inst'])
+    # This class variable can be used when
+    app_target = Target(None)
+
+    @classmethod
+    def is_instance_of(cls, inst, *targets):
+        for target in targets:
+            assert target is None or isinstance(target, cls.Target)
+            if target is None or target.inst is None or target.inst == inst:
+                return True
+        return False
+
+    # Helper type to handle user input
+    Input = namedtuple('Input', ['str', 'is_confirm'])
```

Comment:
Is it required to put this type inside the class?
What kind of dependencies do we have here?
Maybe it is better to make such a construction independent?
What makes `namedtuple` better than a conventional dictionary like `dict[str, bool]`?

Reply:
I've rearranged tests and dropped `Input`.

Reply:
Moved to separate file.